content/downloads/_index.md

```diff
@@ -35,15 +35,46 @@ For full instructions and prerequisites, see [GoFSH Installation](/docs/gofsh/in
 
 ### Presentations
 
-* <a href="https://youtu.be/RfmqpUA606U">Introduction to FHIR Shorthand</a>, from FHIR DevDays, November 2020
-  * Slides <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pdf">here</a>
-* <a href="https://youtu.be/7yzLzQjict0">Let's Build - Profiling with FHIR Shorthand</a>, from FHIR DevDays, November 2020
-  * Slides <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pdf">here</a>
-* <a href="https://youtu.be/Ohgx5TedLlI">Advanced FHIR Shorthand and Tools</a>, from FHIR DevDays, June 2021
-  * Slides <a href="20210610_Chris_Moesel_Advanced_FSH_And_Tools.pdf">here</a>
+* <a href="https://youtu.be/tmtEnjxrCDA">Learn to FSH: A friendly introduction to FHIR Shorthand</a>, from FHIR DevDays, June 2023
+  * Slides <a href="230606_MarkKramer_Learn_to_FSH.pdf">here</a>
+* <a href="https://youtu.be/u6Btqb0Sz3M">Advanced FSH: What to do when you’re hooked</a>, from FHIR DevDays, June 2023
+  * Slides <a href="230609_ChrisMoesel_AdvancedFSH.pdf">here</a>
+* Let's Build - Create an Implementation Guide with FHIR Shorthand, from FHIR DevDays, June 2022
+  * Slides <a href="220607-ChrisMoesel-CreateAnIGWithFHIRShorthand-Chris-Moesel.pdf">here</a>
 * <a href="https://youtu.be/OmcSuAMf20s">Create an Implementation Guide with FHIR Shorthand</a>, from FHIR DevDays, June 2021
   * Slides <a href="20210608_Mark_Kramer_Create_an_IG_with_FSH.pdf">here</a>
+* <a href="https://youtu.be/Ohgx5TedLlI">Advanced FHIR Shorthand and Tools</a>, from FHIR DevDays, June 2021
+  * Slides <a href="20210610_Chris_Moesel_Advanced_FSH_And_Tools.pdf">here</a>
 * <a href="https://youtu.be/0JSp-IOul20">Let's Build - Create an Implementation Guide</a>, from FHIR DevDays, June 2021
   * Slides <a href="20210617_MarkKramer_Lets_Build_Create_an_IG_with_FSH.pdf">here</a>
+* <a href="https://youtu.be/RfmqpUA606U">Introduction to FHIR Shorthand</a>, from FHIR DevDays, November 2020
+  * Slides <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pdf">here</a>
+* <a href="https://youtu.be/7yzLzQjict0">Let's Build - Profiling with FHIR Shorthand</a>, from FHIR DevDays, November 2020
+  * Slides <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pdf">here</a>
+
+### HL7 Community Kickoff and Knowledge Sharing Sessions
+
+The [FSH Tooling Community Kick Off Webinar](https://hl7-org.zoom.us/rec/share/E-adnsXQFT2urCZw3ERdM_HMCXl1PSyBsVwX3h-_SZe9D8uC415UObFdrERYEpa1.av4D1wJRJSCOTmCE) from August 2024 discussed the current state of FHIR Shorthand and its associated tools, the governance of the community-driven project going forward, and possible next steps for the project.
+The [slides](https://confluence.hl7.org/display/FHIR/FSH+Tooling+Community+Kick+Off+Webinar?preview=/265100106/265100122/FSH%20Tooling%20Community%20Kickoff.pdf) can be found on Confluence.
```

Comment:
I noticed that for Confluence, you have linked to the previews rather than the direct PDF downloads. Is that intentional?

Reply:
This feels a little counter intuitive based on my other answer above, but since it was a new section, I didn't mind having the links to Confluence, rather than uploading the PDFs to FSH School. However, the preview was the only link I could find to share the PDF on Confluence. Is there a better way to link to a PDF from Confluence? Or do you prefer the direct download from FSH School?

Reply:
After talking about the options we have for linking to PDFs on Confluence, we decided to keep these preview links as is.